### ISSUE TITLE
Grafana Unexpected "}" in operand

### ISSUE THREAD

**Issue by ben-ole** [helm/charts#1459]
**BUG REPORT**:

**Version of Helm and Kubernetes**:
```
kubectl version
Client Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.6", GitCommit:"7fa1c1756d8bc963f1a389f4a6937dc71f08ada2", GitTreeState:"clean", BuildDate:"2017-06-16T18:34:20Z", GoVersion:"go1.7.6", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.4", GitCommit:"d6f433224538d4f9ca2f7ae19b252e6fcb66a3ae", GitTreeState:"clean", BuildDate:"2017-05-19T18:33:17Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}

$ helm version
Client: &version.Version{SemVer:"v2.5.0", GitCommit:"012cb0ac1a1b2f888144ef5a67b8dab6c2d45be6", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.5.0", GitCommit:"012cb0ac1a1b2f888144ef5a67b8dab6c2d45be6", GitTreeState:"clean"}
```

**Which chart**:
stable/grafana

**What happened**:
running `helm lint` or `helm install` or `helm upgrade` results in

`[ERROR] templates/: parse error in ... template ... /grafana/templates/job.yaml:41: unexpected "}" in operand`

**What you expected to happen**:
no template error

**How to reproduce it** (as minimally and precisely as possible):
Download LATEST commit 912f50c7165383834fc4706ffbe162dd7068b9dd and run `helm lint`. Previously, everything worked, but starting with the new commit (without version change) our deployment pipeline fails.

**Anything else we need to know**:
Would be nice, if you change chart version numbers when updating the charts and run a `helm lint`.

**Comment by chrkaatz** [comment]
just saw the same.

**Comment by patricklucas** [comment]
This will be fixed in #1458.
